frontend/src/component/feature/Dependencies/AddDependencyDialogue.tsx

```diff
@@ -15,21 +16,33 @@ const StyledSelect = styled(GeneralSelect)(({ theme }) => ({
     marginBottom: theme.spacing(1.5),
 }));
 
+const REMOVE_DEPENDENCY_OPTION = {
```

Comment:
"remove" option could be more visually distinct 
https://mui.com/material-ui/react-select/#labels-and-helper-text

Reply:
Will address in the next PR

Reply:
I checked this and we use GeneralSelect that unfortunately uses configuration over composition approach. It prevents styling of children. Since this component is temporary and we'll have separate remove dependency widget in future I'll leave it for now.